cmd/mavenBuild.go

```diff
@@ -85,20 +85,34 @@ func runMavenBuild(config *mavenBuildOptions, telemetryData *telemetry.CustomDat
 
 	_, err := maven.Execute(&mavenOptions, utils)
 
-	log.Entry().Infof("creating build settings information...")
+	log.Entry().Debugf("creating build settings information...")
```

Comment:
as above, possible candidate for a re-use function

Reply:
Fix